src/pages/explorer/Explorer.jsx

```diff
@@ -13,141 +13,235 @@ import { Controlled as CodeMirror } from 'react-codemirror2';
 import { Checkbox, Input, Select, Button, Tooltip, Icon, Tag } from 'antd';
 import 'codemirror/theme/material.css';
 import 'codemirror/lib/codemirror.css';
-import 'codemirror/mode/javascript/javascript'
-import 'codemirror/addon/selection/active-line.js'
-import 'codemirror/addon/edit/matchbrackets.js'
-import 'codemirror/addon/edit/closebrackets.js'
-import "./explorer.css"
-import { notify } from "../../utils";
+import 'codemirror/mode/javascript/javascript';
+import 'codemirror/addon/selection/active-line.js';
+import 'codemirror/addon/edit/matchbrackets.js';
+import 'codemirror/addon/edit/closebrackets.js';
+import './explorer.css';
+import { notify } from '../../utils';
+import GraphiQL from 'graphiql';
+import fetch from 'isomorphic-fetch';
+import 'graphiql/graphiql.css';
 
 const { Option } = Select;
 
+const generateAdminToken = secret => {
+  return jwt.sign({ id: SPACE_CLOUD_USER_ID }, secret);
+};
 
-const generateAdminToken = (secret) => {
-  return jwt.sign({ id: SPACE_CLOUD_USER_ID }, secret)
-}
+function graphQLFetcher(graphQLParams, project) {
+
+  return fetch(`/v1/api/graphql/${project}`, {
+    method: 'post',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify(graphQLParams)
+  }).then(response => response.json());
 
-const Explorer = (props) => {
-  const [value, setValue] = useState(templates.defaultTemplate)
-  const [useAdminToken, setUseAdminToken] = useState(true)
-  const [token, setToken] = useState(null)
-  const [loading, setLoading] = useState(null)
-  const [response, setResponse] = useState(null)
-  const [selectedTemplate, setSelectedTemplate] = useState()
+}
 
+const Explorer = props => {
+  const [useAdminToken, setUseAdminToken] = useState(true);
```

Comment:
Keep `useAdminToken` and `token` in redux as well under the `uiState.explorer.spaceApi` and  `uiState.explorer.graphql` fields. useAdminToken and token will come in both GraphQL and SpaceAPI explorer.